src/locale/ja.js

```diff
@@ -19,6 +19,7 @@ const locale = {
     lll: 'YYYY年M月D日 HH:mm',
     llll: 'YYYY年M月D日(ddd) HH:mm'
   },
```

Comment:
is it evey locale have this method ?

Reply:
if locale need other than am/pm might need this. moment.js'locale file is a good reference

Reply:
lgtm, if we have updated documentation too 👍